cmd/api/src/api/v2/agi.go

```diff
@@ -430,6 +430,49 @@ func (s Resources) ListAssetGroupMembers(response http.ResponseWriter, request *
 	}
 }
 
+func (s Resources) ListAssetGroupMemberCountsByKind(response http.ResponseWriter, request *http.Request) {
```

Comment:
This uses a lot of the above list assetgroupmember fn, did we want to abstract this out a bit or good to keep the barriers up across endpoints?